personal-brand/personal-branding.md

```diff
@@ -27,9 +27,8 @@
 
 ### Materials
 
-- [templates.netlify.com](http://templates.netlify.com/)
-- [codenewbie](https://www.codenewbie.org/)
 - [How to win friends and stand out from the crowd](https://pizza.netlify.com)
+- [Tutorial](https://blog.markgrowth.com/how-to-start-a-beautiful-personal-blog-using-medium-ca6d89b7e46)-Create blog on Medium.
```

Comment:
Can you help us by noting what these three changed resources are next to each link?

Reply:
@alodahl, Shall I add description next to each link. I am not clear with what you meant by "Can you help us by noting what these three changed resources are next to each link?"